marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mappingWithCustomHeader.cy.tsx

```diff
@@ -150,7 +150,6 @@ describe("Create and verify load steps, map step and flows with a custom header"
     cy.waitForAsyncRequest();
     cy.waitUntil(() => curatePage.getEntityTypePanel("Order").should("be.visible").click());
 
-    // TODO: Need to wait do to a second re-render. BUG: DHFPROD-9222
```

Comment:
Do you know if this bug is still happening here? I saw the ticket was rejected saying that is doesn't happen.